texastribune/donations-app
# Striped

#### What's this PR do?

Updates Givalike urls -> Stripe form urls, in JS for top and bottom checkouts as well as for form links throughout the app.
#### Why are we doing this? How does it help us?

Helps us with our switch from Givalike -> Stripe
#### How should this be manually tested?

Run the project locally.

On the index page, /, check out the links in the top checkout and see that "Join or Renew" and "One-Time Donation" both are linking to the form that matches up with the new Stripe forms, and that the query params also match up with what those forms are expecting. Also check the links in the right sidebar for each level are linking where you'd expect for the forms with the correct query params (should either go to memberform or circleform, and should pass through amount, installmentPeriod, and installments if circle). Look at the bottom checkout links, as well. Check the "Donate" link in the footer. 

On the /levels.html page, check that the links for levels in the grid link out where you'd expect them to, to the correct forms with the appropriate query params.

On the /circle.html page, check that "Join" and "Renew" both link out to the circleform, with query params for amount, installmentPeriod, and installments.
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?
#### What are the relevant tickets?

Part of Sprint task.
#### Next steps?
#### Smells?
#### TODOs:
- [x] Update urls to not include support.tt.org where appropriate
- [x] Update urls for membership, donate and circle to point to correct forms
- [x] Wait for Stripe switchover to happen
#### Has the relevant documentation/wiki been updated?

Will update when merged.
#### Technical debt note

Same
